patches/chrome-browser-ui-tabs-tab_style.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/tabs/tab_style.cc b/chrome/browser/ui/tabs/tab_style.cc
+index aa9cd303743116892d65f71f66759bc4cefe05f5..25a252b713af07ecc4cf3ecd0312d48b9f9eba3a 100644
+--- a/chrome/browser/ui/tabs/tab_style.cc
++++ b/chrome/browser/ui/tabs/tab_style.cc
+@@ -275,6 +275,7 @@ SkColor ChromeRefresh2023TabStyle::GetTabBackgroundColor(
+ 
+ // static
+ const TabStyle* TabStyle::Get() {
++  BRAVE_TAB_STYLE_GET
```

Comment:
@mkarolin this looks doable with a chromium_src override on `Get` or am I missing something that required a patch here?

Reply:
oh didn't realize this was so old, but still...